### PR TITLE
Character

### PR DESCRIPTION
**Pull request by CJPoll**
First stab at creating a character entity.

Discussion welcome.

### CODE REVIEW

File: src/character.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    def addXP(self, amount):
+        self.xp += amount
+
+    def baseStat(self, stat):
+        return self.baseStats[stat]
+
+    def buffStat(self, stat, amount):
+        self.statBuffs[stat] += amount
+
+    def buffsOn(self, stat):
```
Comment (thebriz24) [inline src/character.py]: I'm not sure what I want this name to be. I'd say statBuffs but that'd be a bit confusing with buffStats above this.

**Review by thebriz24 — COMMENTED** [review]
Maybe a little something. If we don't feel like changing it that's fine too and I'll approve it.

File: src/character.py (hunk 1 above)
Comment (CJPoll) [reply src/character.py]: _exactly_ what I thought

Comment (CJPoll) [reply src/character.py]: `buffsOn("body")` is the current form - I'm open to suggestions.

**Comment by thebriz24** [comment]
Maybe whichBuffOn("body")? Or quantity?

**Review by thebriz24 — APPROVED** [review]
Solid

**Pull request merged**